websites/management/commands/markdown_cleaning/removal_rules.py

```diff
@@ -0,0 +1,22 @@
+from websites.management.commands.markdown_cleaning.cleanup_rule import (
+    RegexpCleanupRule
+)
+
+class RemoveInaccesibleGif(RegexpCleanupRule):
+    """
+    Remove all instances of "![This resource may not render correctly in a screen reader.](/images/inacessible.gif)".
+    """
+    alias = "inaccessible"
+
+    # Yes, literally this. Period.
+    # Helpfully, inacessible is mispelled which makes it even more specific!.
+    regex=r'!\[This resource may not render correctly in a screen reader\.\]\(/images/inacessible\.gif\)'
```

Comment:
All instances have exactly this form.

There are 7219 instances of "inacessible" in OCW content, and 7219 matches to this regex. Helpfully, the image has "inaccessible" misspelled, else we'd probably get more matches to the word query.